outlines/exploring-graphiql-the-in-browser-ide-for-graphql.md

```diff
@@ -0,0 +1,67 @@
+# Exploring GraphiQL: The In-Browser IDE for GraphQL
+
+**Target Audience:** This article is for all developers and testers who integrate or test GraphQL APIs, whether front-end developers, back-end developers, or QA testers. It's also for anyone else interested in the world of GraphQL APIs.
+
+**Aim:** This article aims to provide developers with an in-depth understanding of GraphiQL, its features, and how to effectively use it to interact with GraphQL APIs. It will cover the fundamentals, and practical applications for utilizing GraphiQL as an in-browser IDE for GraphQL.
+
+## Introduction
+Why waste time and effort troubleshooting your GraphQL API directly in your application when there's a more efficient way? Imagine testing and perfecting your queries outside your app, ensuring they work flawlessly before implementation. Sounds ideal, right? With [GraphiQL](https://az.dev/swapi-graphql), you're saving time and gaining control over your development process, empowering you to work more confidently and effectively.
+
+## Prerequisites
+To follow along with this guide, you need:
+* Fundamental understanding of GraphQL 
+* Basic knowledge of schemas, queries, mutations, and resolvers. 
+* A web browser
+* A GraphQL API endpoint: This is how GraphiQL can communicate with a server to access the data. 
+
+## What is GraphiQL?
+* Definition and overview of GraphiQL
+
+## Why use GraphiQL?
+* Advantages/benefits of using GraphiQL (things like Improving your request, validating your improvements, and easily debugging requests running into problems)
+
+## Methods for accessing GraphiQL
+To access GraphiQL, you are not limited to the default in-browser method. 
+* Head to [https://graphql.org/swapi-graphql](https://graphql.org/swapi-graphql) by querying your search engine (browser) to access the GraphiQL online editor, which works with the Star Wars data, and it is publicly available for you to use
+* If you are using Gatsby, you can start the Gatsby development server by pasting this: `http://localhost:8000/___graphql` into your browser
```

Comment:
```suggestion
* If you are using Gatsby, you can start the Gatsby development server by pasting this: `http://localhost:8000/___graphql` into your browser
```
This should reference tailcall

Reply:
How do I do this? Is it the Tailcall website I am referencing?

Reply:
check this out https://tailcall.run/docs/#starting-the-graphql-server